model/src/main/kotlin/Error.kt

```diff
@@ -20,10 +20,15 @@
 package com.here.ort.model
```

Comment:
Commit message nit: "converts". Also, the commit message should probably briefly say *why* we do this: "... this is to normalize error messages across platforms to easily compare them to expected results..." or so.

Reply:
Done.